osu.Game/Screens/Select/Details.cs

```diff
@@ -0,0 +1,545 @@
+                                            Origin = Anchor.TopCentre,
+                                        },
+                                        ratingsBar = new DetailsBar
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            Height = 5,
+                                            Length = 0,
+                                        },
+                                        new Container
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            AutoSizeAxes = Axes.Y,
+                                            Children = new[]
+                                            {
+                                                negativeRatings = new SpriteText
+                                                {
+                                                    TextSize = 14,
+                                                    Font = @"Exo2.0-Medium",
+                                                    Text = "0",
+                                                },
+                                                positiveRatings = new SpriteText
+                                                {
+                                                    TextSize = 14,
+                                                    Font = @"Exo2.0-Medium",
+                                                    Text = "0",
+                                                    Anchor = Anchor.TopRight,
+                                                    Origin = Anchor.TopRight,
+                                                },
+                                            },
+                                        },
+                                        new SpriteText
+                                        {
+                                            Text = "Rating Spread",
+                                            TextSize = 14,
+                                            Font = @"Exo2.0-Medium",
+                                            Anchor = Anchor.TopCentre,
+                                            Origin = Anchor.TopCentre,
+                                        },
+                                        ratingsGraph = new FillFlowContainer<DetailsBar>
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            Direction = FillDirection.Horizontal,
+                                            Height = 50,
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+                new FillFlowContainer

```

Comment:
The bar graph here should be aligned just below the content before this container. So when the panel expands the graph remains in the same position but the background extends vertically as it currently does.